flags.go

```diff
@@ -267,6 +267,13 @@ func newApp() (app *cli.App) {
 					" in the meantime using the same mount, since we are not refreshing the cache, it will still return nil.",
 			},
 
+			cli.IntFlag{
+				Name:  "grpc-conn-pool-size",
+				Value: 4,
+				Usage: "Number of gRPC channel. Normally, it's same as number of tcp connection, but in case of" +
+					"direct-path traffic it's hard to determine number of tcp-connections in one grpc channel.",
```

Comment:
the description is not clear. Are you saying one tcp connection is same as one gRPC channel?

Reply:
Yes, for cloud-path number of gRPC channel is same as number of tcp connections.

Reply:
what is a good value for direct-path?